scripts/canisters/ic_deploy/upgrade_all.sh

```diff
@@ -4,11 +4,11 @@ set -euo pipefail
 dfx build --network=ic individual_user_template
 gzip -f -1 ./target/wasm32-unknown-unknown/release/individual_user_template.wasm
 dfx build --network=ic user_index
-dfx build --network=ic configuration
+# dfx build --network=ic configuration
 # dfx build --network=ic data_backup
 dfx build --network=ic post_cache
 
-dfx canister install configuration --network ic --mode upgrade --argument "(record {})"
+# dfx canister install configuration --network ic --mode upgrade --argument "(record {})"
```

Comment:
Lets remove it completely instead of commenting

Reply:
Done